config/locales/en.yml

```diff
@@ -2883,6 +2883,7 @@ en:
         reactivate: Reactivate
         comment_and_resolve: Comment & Resolve
         comment: Comment
+        report_link_html: "If this note contains sensitive information that needs to be removed by an admin, you can %{link}. For all other cases, please resolve the note yourself with a comment"
```

Comment:
This should contain a trailing period. Additionally, I'd remove "by an admin" as it doesn't matter to the user who removes it.